resources/scenarios/ln_init.py

```
@@ -6,6 +6,8 @@
 from commander import Commander
 from ln_framework.ln import Policy
 
+THREAD_JOIN_TIMEOUT = 20
```

Im not sure I agree with this, for example if a channel doesn't open within 20 seconds, the scenario will just silently continue, and the user won't have the network graph they wanted. Were you finding some operations to hang longer than 20 seconds?

---

removed